manifest_staging/charts/csi-secrets-store-provider-azure/templates/arc-monitoring-config.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.enableArcExtension }}
+{{- if .Values.arc.enableMonitoring }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: arc-telegraf-config
+  namespace: {{ .Release.Namespace }}
```

Comment:
Should we add `sscdpa.arc.labels`  for all the configmaps too?

Reply:
Added.